pyFAST/input_output/tests/example_files/FASTIn_TurbSim.inp

```diff
@@ -0,0 +1,74 @@
+"A"       IECturbc        - IEC turbulence characteristic ("A", "B", "C" or the turbulence intensity in percent) ("KHTEST" option with NWTCUP model, not used for other models)
+"NTM"     IEC_WindType    - IEC turbulence type ("NTM"=normal, "xETM"=extreme turbulence, "xEWM1"=extreme 1-year wind, "xEWM50"=extreme 50-year wind, where x=wind turbine class 1, 2, or 3)
+default   ETMc            - IEC Extreme Turbulence Model "c" parameter [m/s]
+"PL"      WindProfileType - Velocity profile type ("LOG";"PL"=power law;"JET";"H2L"=Log law for TIDAL model;"API";"USR";"TS";"IEC"=PL on rotor disk, LOG elsewhere; or "default")
+"unused"  ProfileFile     - Name of the file that contains input profiles for WindProfileType="USR" and/or TurbModel="USRVKM" [-]
+84.2876   RefHt           - Height of the reference velocity (URef) [m]
+18.2      URef            - Mean (total) velocity at the reference height [m/s] (or "default" for JET velocity profile) [must be 1-hr mean for API model; otherwise is the mean over AnalysisTime seconds]
+default   ZJetMax         - Jet height [m] (used only for JET velocity profile, valid 70-490 m)
+default   PLExp           - Power law exponent [-] (or "default")
+default   Z0              - Surface roughness length [m] (or "default")
+
+--------Non-IEC Meteorological Boundary Conditions------------
+default   Latitude        - Site latitude [degrees] (or "default")
+0.05      RICH_NO         - Gradient Richardson number [-]
+default   UStar           - Friction or shear velocity [m/s] (or "default")
+default   ZI              - Mixing layer depth [m] (or "default")
+default   PC_UW           - Hub mean u'w' Reynolds stress [m^2/s^2] (or "default" or "none")
+default   PC_UV           - Hub mean u'v' Reynolds stress [m^2/s^2] (or "default" or "none")
+default   PC_VW           - Hub mean v'w' Reynolds stress [m^2/s^2] (or "default" or "none")
+
+--------Spatial Coherence Parameters----------------------------
+default   SCMod1           - u-component coherence model ("GENERAL", "IEC", "API", "NONE", or "default")
+default   SCMod2           - v-component coherence model ("GENERAL", "IEC", "NONE", or "default")
+default   SCMod3           - w-component coherence model ("GENERAL", "IEC", "NONE", or "default")
+default   InCDec1          - u-component coherence parameters for general or IEC models [-, m^-1] (e.g. "10.0  0.3e-3" in quotes) (or "default")
+default   InCDec2          - v-component coherence parameters for general or IEC models [-, m^-1] (e.g. "10.0  0.3e-3" in quotes) (or "default")
+default   InCDec3          - w-component coherence parameters for general or IEC models [-, m^-1] (e.g. "10.0  0.3e-3" in quotes) (or "default")
+default   CohExp           - Coherence exponent for general model [-] (or "default")
+
+--------Coherent Turbulence Scaling Parameters-------------------
+"C:/Yiwork_NREL/OffshoreWind/Turbsim/Test/EventData"  CTEventPath     - Name of the path where event data files are located

```

Comment:
You might want to replace the path here with a generic path. `Test/EventData`

Reply:
Thanks for your suggestion!  You're very careful and meticulous.